### PR TITLE
AngularJS 0.6.4: Meta Improvements

### PR DESCRIPTION
**Pull request by alexgurrola**
Changes:

- Update `meta.get()` Functionality

Fixes:

- Fatal on Malformatted Meta

### CODE REVIEW

**Review by apocist — APPROVED** [review]

**Comment by alexgurrola** [comment]
Note: This change relies on the changes in https://github.com/Sitetheory/stratus/pull/739.

File: packages/angularjs/src/services/model.ts [hunk 1]
```diff
@@ -614,7 +614,7 @@ export class Model<T = LooseObject> extends ModelBase<T> {
                 const status: { code: string }[] = this.meta.get('status') || []
 
                 // Evaluate Payload
-                if (this.meta.has('status') && _.first(status).code !== 'SUCCESS') {
+                if (this.meta.get('status[0].code') !== 'SUCCESS') {
```
Comment (apocist) [inline packages/angularjs/src/services/model.ts:617]: This seems to now break any Model fetchs when we have no expectation for there to a return meta or meta.status

Comment (alexgurrola) [reply packages/angularjs/src/services/model.ts:617]: The logic was almost entirely the same except for the removal of the `this.meta.has('status')` conditional, so I'll submit a patch with that addition shortly.  The other addition of `meta.get()` is contingent on the `@stratusjs/core` with a version of  `0.6.0`+, but that doesn't appear to a problem in your install.

Comment (alexgurrola) [reply packages/angularjs/src/services/model.ts:617]: This has been fixed in https://github.com/Sitetheory/stratus/pull/741.

**Pull request merged**